carta/session.py

```diff
@@ -383,6 +383,19 @@ def append_image(self, path, hdu=""):
         """
         return Image.new(self, path, hdu, True)
 
+    def open_images(self, *paths):
+        """Open multiple images.
+
+        Parameters
+        ----------
+        paths : {0}
+            The paths to the image files, either relative to the session's current directory or an absolute path relative to the CARTA backend's root directory. Separate them with comma.
```

Comment:
`an absolute path` -> `absolute paths`

This is *not* a comma-separated string; it's an iterable of strings. Separate parameters passed to this function will be stored in a single tuple parameter called `paths`. (Look up `python args kwargs` to see how these "magic" variables work -- the names are conventions; it's fine to use the name `paths` here.)